TLM/TLM/LoadingExtension.cs

```diff
@@ -1,465 +1,584 @@
+    using System.Reflection;
     using API.Manager;
-
+    using ColossalFramework;
+    using ColossalFramework.UI;
+    using CSUtil.Commons;
+    using Custom.PathFinding;
+    using Harmony;
+    using ICities;
+    using JetBrains.Annotations;
+    using Manager.Impl;
+    using RedirectionFramework;
+    using State;
+    using UI;
+    using Object = UnityEngine.Object;
+
+    [UsedImplicitly]
     public class LoadingExtension : LoadingExtensionBase {
-		private const string HARMONY_ID = "de.viathinksoft.tmpe";
-
-		public class Detour {
-			public MethodInfo OriginalMethod;
-			public MethodInfo CustomMethod;
-			public RedirectCallsState Redirect;
-
-			public Detour(MethodInfo originalMethod, MethodInfo customMethod) {
-				this.OriginalMethod = originalMethod;
-				this.CustomMethod = customMethod;
-				this.Redirect = RedirectionHelper.RedirectCalls(originalMethod, customMethod);
-			}
-		}
-
-		public class ManualHarmonyPatch {
-			public MethodInfo method = null;
-			public HarmonyMethod prefix = null;
-			public HarmonyMethod transpiler = null;
-			public HarmonyMethod postfix = null;
-		}
-
-		//public static LoadingExtension Instance;
-
-		public static bool IsPathManagerReplaced {
-			get; private set;
-		} = false;
-
-		public static CustomPathManager CustomPathManager { get; set; }
-		public static bool DetourInited { get; set; }
-		public static List<Detour> Detours { get; set; }
-		public static HarmonyInstance HarmonyInst { get; private set; }
-		//public static TrafficManagerMode ToolMode { get; set; }
-		//public static TrafficManagerTool TrafficManagerTool { get; set; }
+        private const string HARMONY_ID = "de.viathinksoft.tmpe";
+
+        public class Detour {
+            public MethodInfo OriginalMethod;
+            public MethodInfo CustomMethod;
+            public RedirectCallsState Redirect;
+
+            public Detour(MethodInfo originalMethod, MethodInfo customMethod) {
+                OriginalMethod = originalMethod;
+                CustomMethod = customMethod;
+                Redirect = RedirectionHelper.RedirectCalls(originalMethod, customMethod);
+            }
+        }
+
+        public class ManualHarmonyPatch {
+            public MethodInfo method;
+            public HarmonyMethod prefix;
+            public HarmonyMethod transpiler = null;
+            public HarmonyMethod postfix = null;
+        }
+
+        // public static LoadingExtension Instance;
+
+        public static bool IsPathManagerReplaced {
+            get; private set;
+        }
+
+        public static CustomPathManager CustomPathManager { get; set; }
+
+        public static bool DetourInited { get; set; }
+
+        public static List<Detour> Detours { get; set; }
+
+        public static HarmonyInstance HarmonyInst { get; private set; }
+        // public static TrafficManagerMode ToolMode { get; set; }
+        // public static TrafficManagerTool TrafficManagerTool { get; set; }
 #if !TAM
```

Comment:
I think this might be obsolete - [TAM](https://steamcommunity.com/sharedfiles/filedetails/?id=543703997) was a mod from some years back that never got off the ground. It's traffic components were merged back in to TM:PE and its roads went on to become Network Extensions 2.

There is also a reference to `TAM`, likely obsolete, in `ThreadingExtension.cs`

Reply:
Removed